tests/tasks/setup/eks/awscli-cfn-lt-al2023.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template-al2023
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS CFN stack to output a launch template for AL2023-based nodes.
+    This Task can be used to create an EKS CFN stack that outputs a launch template.
+    The launch template may be used for a managed nodegroup with or without a custom AMI.
+  params:
+    - name: cluster-name
+      description: EKS cluster you want to create CFN stack for.
+    - name: stack-name
+      description: Stack name you want to spin.
+    - name: region
+      default: "us-west-2"
+      description: The region where the cluster is in.
+    - name: kubernetes-version
+      default: "1.28"
```

Comment:
let's use the latest k8s version as default please.

Reply:
Sure, 1.28 is in the other existing Task FYI.

Reply:
Yeah, those are old ones :D, should be updated. Trying to avoid it at least for new ones :D